packages/babel-plugin-jest-unmock/src/index.js

```diff
@@ -9,31 +9,46 @@
 'use strict';
 
 const JEST_GLOBAL = {name: 'jest'};
-const UNMOCK_FN = {name: 'unmock'};
+
+const FUNCTIONS = {
+  mock: {
+    checkArgs: (args) => args.length === 1 && args[0].isStringLiteral(),
```

Comment:
can you remove the parenthesis around the `args`. The code style at FB is (mostly, but certainly in Jest) to not include them for single params.